LogToDB.cs

```diff
@@ -1,35 +1,94 @@
+            DBconnection db = new DBconnection();
+            if (db.InsertQuery(Logging(type, message, username)) != -1)
+            {
+                db.Close();
+            }
+            else
+            {
+                var logToFile = new LogToFile();
+                logToFile.Error("Не удалось подключиться к БД");
+            }
         }
-        public string Error(string message, string username)
+        public void Info(string message, string username)
         {
-            return Log("ERROR", username, message);
+            DBconnection db = new DBconnection();
+            if (db.InsertQuery(Logging("INFO", message, username)) != -1)
+            {
+                db.Close();
+            }
+            else
+            {
+                var logToFile = new LogToFile();
+                logToFile.Error("Не удалось подключиться к БД");
+            }
         }
-        public string Warning(string message, string username)
+        public void Error(string message, string username)
         {
-            return Log("WARNING", username, message);
+            DBconnection db = new DBconnection();
+            if (db.InsertQuery(Logging("ERROR", message, username)) != -1)
+            {
+                db.Close();
+            }
+            else
+            {
+                var logToFile = new LogToFile();
+                logToFile.Error("Не удалось подключиться к БД");
+            }
         }
-        public string Fatal(string message, string username)
+        public void Warning(string message, string username)
         {
-            return Log("FATAL", username, message);
+            DBconnection db = new DBconnection();
+            if (db.InsertQuery(Logging("WARNING", message, username)) != -1)
+            {
+                db.Close();
+            }
+            else
+            {
+                var logToFile = new LogToFile();
+                logToFile.Error("Не удалось подключиться к БД");
+            }
         }
-        public string Success(string message, string username)
+        public void Fatal(string message, string username)
         {
-            return Log("SUCCES", username, message);
+            DBconnection db = new DBconnection();
+            if (db.InsertQuery(Logging("FATAL", message, username)) != -1)
+            {
+                db.Close();
+            }
+            else
+            {
+                var logToFile = new LogToFile();
+                logToFile.Fatal(Logging("FATAL", username, message));
+            }
+        }
+        public void Success(string message, string username)
+        {
+            DBconnection db = new DBconnection();
+            if (db.InsertQuery(Logging("SUCCESS",message,username)) != -1)
+            {
+                db.Close();
+            }
+            else
+            {
+                var logToFile = new LogToFile();
+                logToFile.Error("Не удалось подключиться к БД");
```

Comment:
Лучше сей момент реализовать через делегаты/эвенты, тогда программист сам определит стоит ли записывать куда-то сообщения или нет.
```suggestion
                logToFile.Error("Не удалось подключиться к БД");
```